clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -735,7 +735,7 @@ private void initializeIsolatedIngestionServer() {
         storeVersionStateSerializer,
         partitionStateSerializer,
         storeRepository,
-        false,
+        !isDaVinciClient,
```

Comment:
So there is no data partition leaking issue in DaVinci?

Reply:
The issue is only triggered by Helix issuing OFFLINE->DROPPED message upon server restart. It's likely because the version becomes stale when server is down/being restarted. The case is not frequent, and this case should not happen to DVC.

Reply:
Got it, and let us add these statements as javadoc.

Reply:
Sure, updated the statement.